@app/db/package.json

```diff
@@ -3,21 +3,22 @@
   "private": true,
   "version": "0.0.0",
   "scripts": {
-    "gm": "node -r @app/config/env node_modules/.bin/graphile-migrate",
+    "gm": "npx -n \"-r @app/config/env\" graphile-migrate",
     "migrate": "yarn gm migrate",
     "watch": "yarn gm watch",
     "commit": "yarn gm commit",
     "uncommit": "yarn gm uncommit",
     "reset": "yarn gm reset",
     "dump": "yarn gm migrate && yarn gm reset --shadow --erase && yarn gm migrate --shadow --forceActions",
     "wipe-if-demo": "./scripts/wipe-if-demo",
-    "test": "NODE_ENV=test node -r @app/config/env ./node_modules/.bin/jest"
+    "test": "cross-env NODE_ENV=test npx -n \"-r @app/config/env\" jest"
```

Comment:
I wish there was a more efficient way to support Windows; at the moment this is spawning a Node process `cross-env` which spawns a Node process `npx` which then spawns _another_ Node process `jest`. So we've got a stack of 3 node processes, where previously we only had one. Is there a particular reason we need to use `npx` here for Windows support?

Reply:
Unfortunately, I don't know a better way to handle these test cases on Windows, as we need to set environment variable, and only using `npx` can find the proper `jest` command on Windows.